File: androidApp/src/main/java/com/mrebollob/loteria/android/presentation/home/HomeUiState.kt
package com.mrebollob.loteria.android.presentation.home

import com.mrebollob.loteria.android.presentation.platform.ErrorMessage
import com.mrebollob.loteria.domain.entity.Ticket
import java.util.Date

data class HomeUiState(
    val today: Date,
    val daysToLotteryDraw: Int,
    val totalBet: Float,
    val totalPrize: Float?,
    val tickets: List<Ticket>,
    val isLoading: Boolean,
    val errorMessages: List<ErrorMessage>,
)

File: iosApp/iosApp/Presentation/Home/HomeUiState.swift
import Foundation
import shared

struct HomeUiState {
    var today: Date =  Date()
    var daysToLotteryDraw: Int = 0
    var totalBet: Float = 0
    var totalPrize: Float? = nil
    var tickets: [Ticket] = []
    var isLoading: Bool = false
    var errorMessages: [String] = []
}
